main.go

```diff
@@ -82,7 +87,7 @@ func partitionIn(services []*string, size int) [][]*string {
 		slice := services[startRange:endRange]
 		allSlices = append(allSlices, slice)
 		startRange = endRange
-		endRange += 10
+		endRange += size
```

Comment:
This will be limited by  the`maxServicesPerRequest`, so you can remove this param and make this function even simpler:

```suggestion
		endRange += maxServicesPerRequest
```